bento-compose/src/main/java/com/yelp/android/bento/compose/ComposeViewHolder.kt

```diff
@@ -0,0 +1,52 @@
+package com.yelp.android.bento.compose
+
+import android.view.View
+import android.view.ViewGroup
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.platform.ComposeView
+import androidx.compose.ui.platform.ViewCompositionStrategy
+import com.yelp.android.bento.core.ComponentViewHolder
+
+/**
+ * ViewHolder which allows compatability with Jetpack Compose. Basically, this lets you write the
+ * view holders view code with Compose.
+ */
+abstract class ComposeViewHolder<P, T> : ComponentViewHolder<P, T>() {
```

Comment:
Q: we will be using AutoMviComposeViewHolder for MVI flow and use this ComposeViewHolder for non-MVI, right?

Reply:
Correct.